cmd/node/main.go

```diff
@@ -135,7 +103,27 @@ func main() {
 	log.Printf("WALDir: %s", *walDir)
 	log.Printf("Members: %s", *members)
 	log.Printf("KeyRange: %s", *keyrange)
-	
+
+	// If not initialized, wait for all nodes and propose initial key ranges
+	if !initialized {
```

Comment:
Are we waiting for all the nodes within the raft group to start here?

Reply:
yes. We assert Raft group is ready in RAFT_LAUNCH_TIMEOUT and then propose the initial key ranges.

Reply:
How do you make sure that all nodes have started? I guess they are all expected to start within `RAFT_LAUNCH_TIMEOUT` but it's hard to guarantee that right?

Reply:
The answer is no loll. Just a rough estimation.

Reply:
we should add some mechanism to let adaptodb know if raft groups are ready.

Reply:
That's a good idea